Makefile

```diff
@@ -41,7 +41,7 @@ ifeq ($(HAZRUST),yes)
 .PHONY: jsoncdc.so
 jsoncdc.so:
 	cargo rustc $(FEATURES) --release -- $(LINK_FLAGS)
-	cp target/release/libjsoncdc.* $@
+	cp target/release/libjsoncdc.*so $@
```

Comment:
Breaks on my Mac (where it is `libjsoncdc.dylib`). What lead to this change?

Reply:
Ahh, this no longer works on linux as two files match libjsoncdc.*